server.js

```diff
@@ -6,6 +6,7 @@ var express = require('express'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
+    require('dotenv').config();
 
 /**
  * Main application entry file.
```

Comment:
Yes. That was because I cloned Esther's branch. It was part of what you asked her to change during this morning session.